src/utils.ts

```diff
@@ -1,13 +1,17 @@
-const regex = /^v(\d+)\.(\d+)\.(\d+)$/g;
+const regex = /^v(\d+)\.(\d+)\.(\d+)$/;
 
 export const alsSupported = () =>
   nodeVersionGreaterThanEqual('v12.17.0') ||
   nodeVersionGreaterThanEqual('v13.14.0') ||
   nodeVersionGreaterThanEqual('v14.0.0');
 
 export const nodeVersionGreaterThanEqual = (requestedVersion: string, version = process.version) => {
-  const [major, minor, patch] = [...version.matchAll(regex)][0].slice(1).map((n: string) => parseInt(n, 10));
-  const [requestedMajor, requestedMinor, requestedPatch] = [...requestedVersion.matchAll(regex)][0]
+  const [major, minor, patch] = version
+    .match(regex)
+    .slice(1)
+    .map((n: string) => parseInt(n, 10));
```

Comment:
👏 Nice! 
nit: 
```suggestion
  const [, major, minor, patch] = version
    .match(regex)
    .map((n: string) => parseInt(n, 10));
```

Reply:
Done. @klesgidis we have one more change that needs to be done in order to support Node10 too. We need an alternative to Object.fromEntries. See https://github.com/feross/fromentries/blob/master/index.js